benchmarks/CT14_bench.py

```diff
@@ -62,15 +63,16 @@ def benchmark_llo_NF3(self, Q0=5, Q2grid=(100,)):
             "T15",
             "V15",
         ]
-        self.run([theory_card], [operator_card], ["CT14llo_NF3"])
+        new_theory, new_operators = compatibility.update(theory_card, operator_card)
```

Comment:
this has to be lifted inside `ekomark` otherwise the others (LHA, etc.) won't benefit from that - more specific it should move just before the `eko` invocation (since the DB is unchanged)